src/integrationtests/resources/com/aws/greengrass/integrationtests/ipc/cda.yaml

```diff
@@ -5,6 +5,8 @@ services:
       runWithDefault:
         posixUser: nobody
         windowsUser: integ-tester
+      logging:
```

Comment:
you can also do `LogConfig.getRootLogConfig().setLevel(Level.DEBUG);` in the test.